json/stage-json/foodPantry.json

```diff
@@ -5242,8 +5234,8 @@
         "password": "",
         "headers": [
           {
-            "header": "",
-            "value": ""
+            "header": "x-token",
+            "value": "ynDAuLoIDSBjPTaZiMQsu1k034LQOO"
```

Comment:
This might have been one of those configurations that fixed something... but I tried adding x-tokens to the config.

Reply:
Do you thing it has something to do with access settings?